app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -25,16 +8,36 @@
 import android.text.Html;
 import android.text.Spanned;
 import android.text.TextUtils;
+import android.text.util.Linkify;
 import android.util.Log;
 
 import com.j256.ormlite.dao.RuntimeExceptionDao;
 import com.j256.ormlite.field.DatabaseField;
 import com.j256.ormlite.misc.TransactionManager;
 import com.j256.ormlite.stmt.DeleteBuilder;
 import com.j256.ormlite.table.DatabaseTable;
+import com.zulip.android.ZulipApp;
 import com.zulip.android.util.CustomHtmlToSpannedConverter;
 import com.zulip.android.util.ZLog;
-import com.zulip.android.ZulipApp;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.ccil.cowan.tagsoup.HTMLSchema;
+import org.ccil.cowan.tagsoup.Parser;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+import static com.zulip.android.util.CustomHtmlToSpannedConverter.linkifySpanned;
```

Comment:
If you're only using this in one location, my preference is that the ref for this stays inline (not imported as a static)